app/code/Magento/Catalog/etc/di.xml

```diff
@@ -1073,4 +1073,9 @@
             <argument name="productRepository" xsi:type="object">Magento\Catalog\Api\ProductRepositoryInterface\Proxy</argument>
         </arguments>
     </type>
+    <type name="Magento\Catalog\Model\Product">
+        <arguments>
+            <argument name="resource" xsi:type="object" shared="false">Magento\Catalog\Model\ResourceModel\Product</argument>
```

Comment:
As product model is quite popular in Magento code such change leads to creating a lot of additional resource models when they are not really necessary.

Please rework as a more isolated fix, most likely as only multiselect attribute type for bundle options is affected it will be there, not in generic product model.

You actually managed to reproduce
> Fixed Issues (if relevant)
magento/magento2#10540
magento/magento2#13520

correct?

Reply:
Correct